client/rpc-servers/Cargo.toml

```
@@ -12,14 +12,16 @@ description = "Substrate RPC servers."
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+futures = "~0.1.6"
```

As far as I can tell we don't use explicit [tilde requirements](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements) in most places. Why use it here in particular?
```suggestion
futures = "0.1.6"
```